docker/README.md

```diff
@@ -0,0 +1,54 @@
+You can use Docker to easily try out or test sendgrid-go.
+
+# Quickstart
+1. Install Docker in your machine.
+2. Run `docker run -it -v /path/to/your-code:/data dhoeric/sendgrid-go`.
+3. Run your code with `go run your-code.go`
+
+# Examples
+1. Copy and save in you local machine, e.g. `test.go`.
+```
+package main
+
+import (
+	"fmt"
+	"github.com/sendgrid/sendgrid-go"
+	"log"
+)
+
+func main() {
+	api_key := "JUST_A_TEST_KEY"
```

Comment:
Rather than having the key embedded in the script, could you have it pull from an environment variable that is loaded into the docker via a docker-compose file? 

This would also allow you to have a "volume" param that would keep the source local for development purposes!